README.md

```diff
@@ -47,9 +47,13 @@ Available Commands:
   update      Update resources
 
 Flags:
-  -h, --help         help for lhctl
-  -t, --toggle       Help message for toggle
-      --url string   longhorn manager url (example: http://10.88.1.3/v1)
+      --config string    config file (default is $HOME/.lhctl.yaml)
+      --context string   config file context
+  -h, --help             help for lhctl
+      --pass string      password for http request
+  -t, --toggle           Help message for toggle
```

Comment:
Outside the scope of this PR but is this flag used? This seems like a placeholder message.

Reply:
yup, you are correct that came with cobra, I shall remove